### ISSUE TITLE
Neither EU or EU2 seem to be working anymore

### ISSUE THREAD

**Issue by thinkpad4** [AlyoshaVasilieva/luminous-ttv#13]
Today neither EU or EU2 are blocking ads anymore.

**Comment by mpsparrow** [comment]
They appear to be down according to the status page https://stats.uptimerobot.com/N2pVRCZAjg

**Comment by AlyoshaVasilieva** [comment]
Twitch finally rolled out its client integrity update* so currently luminous-ttv is broken. I just woke up; I'll look into it but it's entirely possible I won't be able to fix it.

*clients hit some API that verifies that they are real, and must pass that info to Twitch further API interactions. luminous-ttv creates an entirely fake ID by just randomly generating one (to preserve user privacy, avoid needing tokens) which doesn't register as real anymore

**Comment by ketersu** [comment]
vaft script on https://github.com/pixeltris/TwitchAdSolutions seems work now

**Comment by ItzOnlyAnimal** [comment]
Apparently it was rolled back for now. So it may re appear shortly

**Comment by cabbagefork** [comment]
> Apparently twitch's change was rolled back for now. So it may re appear shortly

Can confirm on my end. Updated nothing and works like normal again.

**Comment by thinkpad4** [comment]
I believe Twitch is doing like they did with the Kraken API, a 1 hour shut down, then a 24 hour shutdown (this one), and then permanent shutdown in the near future

**Comment by juliocoliveira** [comment]
I'm using TTV LOL Pro v1.9.2 with https://eu2.luminous.dev server and is working fine.

**Comment by Sokoloft** [comment]
Yes it has been working fine for the past 8 hours or so. However prior the past 24h before that 8hr period they released some update with code from a company called kasada. Maybe next week or next month it will return to being broken until projects are updated. 

https://github.com/streamlink/streamlink/issues/5370#issuecomment-1572883416

**Comment by vinnnyyyyy** [comment]
O Proxys EU e EU2 pararam pra mim :/

**Comment by vinnnyyyyy** [comment]
Proxys EU and EU2 stopped for me, I tested both and they are not working.  :/

**Comment by GloftOfficial** [comment]
> Proxys EU and EU2 stopped for me, I tested both and they are not working. :/

both of them respond correctly on my end . You can check uptime here https://stats.uptimerobot.com/N2pVRCZAjg

**Comment by vinnnyyyyy** [comment]
> > Proxys EU and EU2 stopped for me, I tested both and they are not working. :/
> 
> both of them respond correctly on my end . You can check uptime here https://stats.uptimerobot.com/N2pVRCZAjg

Precisely why I'm finding it strange, apparently only AS is working for me...

**Comment by vinnnyyyyy** [comment]
> > > Proxys EU and EU2 stopped for me, I tested both and they are not working. :/
> > 
> > 
> > both of them respond correctly on my end . You can check uptime here https://stats.uptimerobot.com/N2pVRCZAjg
> 
> Precisely why I'm finding it strange, apparently only AS is working for me...

**Comment by vinnnyyyyy** [comment]
![Captura de tela 2023-06-13 161600](https://github.com/AlyoshaVasilieva/luminous-ttv/assets/135191864/9947596c-33f2-4376-8314-5cf9f9d948bb)

**Comment by juliocoliveira** [comment]
For me is working fine on eu2 + TTV LOL PRO v1.9.3.
Try updating your TTV LOL PRO.
![image](https://github.com/AlyoshaVasilieva/luminous-ttv/assets/6606725/258f864f-c861-4914-a571-4a19e03871a0)

**Comment by vinnnyyyyy** [comment]
> For me is working fine on eu2 + TTV LOL PRO v1.9.3, No pre-rolls or mid-rolls. Try updating your TTV LOL PRO. ![image](https://user-images.githubusercontent.com/6606725/245599131-258f864f-c861-4914-a571-4a19e03871a0.png)

Even updated to V1.2.3 still didn't work.

**Comment by juliocoliveira** [comment]
> > For me is working fine on eu2 + TTV LOL PRO v1.9.3, No pre-rolls or mid-rolls. Try updating your TTV LOL PRO. ![image](https://user-images.githubusercontent.com/6606725/245599131-258f864f-c861-4914-a571-4a19e03871a0.png)
> 
> Even updated to V1.2.3 still didn't work.

1.2.3? No I said v1.9.3, 1.2 is too old.

**Comment by vinnnyyyyy** [comment]
> > > For me is working fine on eu2 + TTV LOL PRO v1.9.3, No pre-rolls or mid-rolls. Try updating your TTV LOL PRO. ![image](https://user-images.githubusercontent.com/6606725/245599131-258f864f-c861-4914-a571-4a19e03871a0.png)
> > 
> > 
> > Even updated to V1.2.3 still didn't work.
> 
> 1.2.3? No I said v1.9.3, 1.2 is too old.

Sorry, I meant v1.9.3* even updated to the latest version of V1 doesn't work for me.

**Comment by juliocoliveira** [comment]
Yeah, Now I'm getting occasional pre-rolls, 1 ad of 14 seconds. Nothing big, I don't seem to get mid-rolls, so far.
For weeks never got one single ad.
The future of proxies/vpn on twitch seem uncertain.

**Comment by thinkpad4** [comment]
@AlyoshaVasilieva any chance of adding any ttvlolpro v2 proxies?  Just wondering.  Thanks

**Comment by AlyoshaVasilieva** [comment]
I started playing BotW and got distracted. I've started getting jumpscared by ads (literally, they show up for <1s, loud) so maybe soon.

**Comment by thinkpad4** [comment]
Anymore thought into creating some ttvlolpro v2 proxies?  Again, just wondering.  Thanks

**Comment by AlyoshaVasilieva** [comment]
I'm seeing a ton of HTTP 403s. Might be rate-limiting bans? My servers only appear to function for ~100 proxy requests after switching IPs before hitting non-stop 403. I think self-hosting should still work, but if this is actually rate-limiting then I don't think public instances will continue to exist.
